database/getData.js

```diff
@@ -0,0 +1,16 @@
+const connection = require('./connect.js');
+
+const getReviews = (params, cb) => {
+  console.log('getData ran');
+  const q = 'SELECT * FROM reviews WHERE listing_id = ?';
```

Comment:
I don't recommend using single letter variable names, even if they're used just once. `query` only takes 4 more letters to write, so just say `query` instead.